src/core/data/store.h

```diff
@@ -329,8 +340,29 @@ class Store {
   bool readable_{false};
   bool writable_{false};
   bool reducible_{false};
+
+ friend class MakeshiftSerializer;
 };
 
+//containts prefix sums for a sub-op
+//to index into its own data
```

Comment:
any vector with the `starts` suffix refers to a prefix sum array. 
ie for sub-task "i" of the fused task, its input stores are located in: `context.inputs[i]:context.inputs[i+1]`